src/Uno.UWP/Devices/Haptics/SimpleHapticsController.Android.cs

```diff
@@ -1,78 +1,83 @@
+        {
+            new SimpleHapticsControllerFeedback(KnownSimpleHapticsControllerWaveforms.Press, TimeSpan.FromMilliseconds(300)),
+            new SimpleHapticsControllerFeedback(KnownSimpleHapticsControllerWaveforms.Click, TimeSpan.FromMilliseconds(100)), // Single Click
+            new SimpleHapticsControllerFeedback(KnownSimpleHapticsControllerWaveforms.Click, TimeSpan.FromMilliseconds(150))  // Double Click
+        };
 
-		public void SendHapticFeedback(SimpleHapticsControllerFeedback feedback)
-		{
-			if (feedback is null)
-			{
-				throw new ArgumentNullException(nameof(feedback));
-			}
+        public void SendHapticFeedback(SimpleHapticsControllerFeedback feedback)
+        {
+            if (feedback is null)
+            {
+                throw new ArgumentNullException(nameof(feedback));
+            }
 
-			if (ContextHelper.Current == null)
-			{
-				throw new InvalidOperationException($"Context must be initialized before {nameof(SendHapticFeedback)} is called.");
-			}
-			try
-			{
-				var activity = (Activity)ContextHelper.Current;
-				var androidFeedback = FeedbackToAndroidFeedback(feedback);
-#pragma warning disable CA1422 // Validate platform compatibility
-				bool hapticFeedbackEnabled = Settings.System.GetInt(activity.ContentResolver, Settings.System.HapticFeedbackEnabled, 0) != 0;
-#pragma warning restore CA1422 // Validate platform compatibility
-				if (hapticFeedbackEnabled)
-				{
-					var executed = activity.Window?.DecorView.PerformHapticFeedback(androidFeedback) ?? false;
-					if (!executed && PhoneVibrationDevice.GetDefault() is { } vibrationDevice)
-					{
-						// Fall back to VibrationDevice
-						vibrationDevice.Vibrate(feedback.Duration);
-					}
-				}
-			}
-			catch (Exception ex)
-			{
-				if (this.Log().IsEnabled(LogLevel.Error))
-				{
-					this.Log().LogError($"Could not send haptic feedback: {ex}");
-				}
-			}
-		}
+            Vibrator vibrator = (Vibrator)Android.App.Application.Context.GetSystemService(Android.Content.Context.VibratorService);
 
-		private static FeedbackConstants FeedbackToAndroidFeedback(SimpleHapticsControllerFeedback feedback)
-		{
-			if (feedback.Waveform == KnownSimpleHapticsControllerWaveforms.Click)
-			{
-				return FeedbackConstants.ContextClick;
-			}
-			else if (feedback.Waveform == KnownSimpleHapticsControllerWaveforms.Press)
-			{
-				return FeedbackConstants.LongPress;
-			}
-			else
-			{
-				throw new NotSupportedException("Unsupported feedback waveform");
-			}
-		}
-	}
+            if (feedback.Waveform == KnownSimpleHapticsControllerWaveforms.Press)
+            {
+                if (vibrator.HasVibrator)
+                {
+                    vibrator.Vibrate((long)feedback.Duration.TotalMilliseconds);
+                }
+                else
+                {
+                    throw new NotSupportedException("Device does not support vibration");
```

Comment:
It's probably best if we skip vibrating altogether. It may be common for simulators to not have this feature. We can probably log a debug message instead.